src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/prisonersearch/PrisonerSearchCustomDimensions.kt

```diff
@@ -1,25 +1,33 @@
 package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.prisonersearch
 
-data class PrisonerSearchCustomDimensions(
```

Comment:
This class has been turned into a simple function below. There is also a similar function for creating the custom dimensions map for the best matched prisoner